doc/_build/html/_static/custom.css

```diff
@@ -0,0 +1,11 @@
+/* Custom CSS properties for sphinx_copybutton */
+button.copybtn {
```

Comment:
This should be added to `doc/_sphinx/theme`, the `_build` directory should be in `.gitignore`

Reply:
moved `custom.css` file form `_build` directory to `doc/_sphinx/theme`
running `make html` command will generate the `custom.css` file into `_build` directory.
Do I need to do something with running `make html` command?

Reply:
The site is built in the pipeline, so you don't have to do anything on your own computer (except testing that it works).
Name the file `copy_button.css` or something like that, so that it is clear what it is for.